testcases/TC06.1-all-pass-4.md

```diff
@@ -3,28 +3,28 @@
 layout: testcase
 
 # Brief, descriptive title for the test case
-title: Meaningful Image - text equivalent in aria-labelledby attribute
+title: Meaningful Image - text equivalent in aria-labelled by attribute
```

Comment:
`aria-labelledby` here references the actual aria attribute, so it should not have a space.

Reply:
fixed